src/TropicalGeometry/linear_space.jl

```diff
@@ -459,35 +441,30 @@ end
 
 Return the tropical matrix used to construct `TropL`.  Raises an error, if it is not cached.
 """
-function tropical_matrix(TropL::TropicalLinearSpace)
+function tropical_matrix(TropL::TropicalLinearSpace{minOrMax}) where minOrMax
```

Comment:
```suggestion
function tropical_matrix(TropL::TropicalLinearSpace{minOrMax,true}) where minOrMax
```

`TropicalLinearSpace` has two parameters (of which the second is currently always `true`).